docs/en/ingest-management/data-streams.asciidoc

```diff
@@ -0,0 +1,105 @@
+[[data-streams]]
+= Data streams
+
+{agent} uses data streams to easily store append-only time series data across multiple indices
+while giving you a single named resource for requests.
+Data streams are well-suited for logs, metrics, traces, and other continuously generated data.
+They offer a host of benefits over other indexing strategies:
+
+* *Reduced number of fields per index*: Indices only need to store a specific subset of your
+data–meaning no more indices with hundreds of thousands of fields.
+This leads to better space efficiency and faster queries.
+As an added bonus, only relevant fields are shown in Discover.
+
+* *More granular data control*: For example, filesystem, load, cpu, network, and process metrics are sent
+to different indices–each potentially with its own rollover, retention, and security permissions.
```

Comment:
```suggestion
to different indices–-each potentially with its own rollover, retention, and security permissions.
```

Reply:
I think -- gets converted into an em dash.

Reply:
That's already an em dash :)

Reply:
It's hard to tell in the source file, but a little easier in the built version:
<img width="685" alt="Screen Shot 2021-03-10 at 7 17 13 PM" src="https://user-images.githubusercontent.com/5618806/110730393-3d378d00-81d5-11eb-9885-9de1a9841939.png">